example/components/linux.py

```diff
@@ -0,0 +1,135 @@
+"""Giraffe component is the API to Giraffe component."""
+from typing import Any, Optional, TextIO, List, NamedTuple
+
+import asyncio
+import click
+import colored
+import contextlib
+import time
+import os
+
+from Octavius.lego.components import RPyCComponent
+
+
+class LinuxRPyCComponent(RPyCComponent):
```

Comment:
Is any of this actually Linux specific?

Reply:
Of curse not, it just an example.
In real life, it will go into "loggableComponent" which "LinuxComponent" and "WindowsComponent" will inherit from.